vite.config.js

```diff
@@ -1,8 +1,9 @@
 import { defineConfig } from 'vite'
 import vue from '@vitejs/plugin-vue'
+import dts from 'vite-plugin-dts'
```

Comment:
Have the vite build generate the typescript dts instead of vue-tsc. I was having problems with vue-tsc and new typescript versions.